google/src/main/scala/org/broadinstitute/dsde/rawls/google/GoogleUtilities.scala

```diff
@@ -36,11 +36,15 @@ trait GoogleUtilities extends LazyLogging with Retry {
     }
   }
 
-  protected def retryWhen500orGoogleError[T](op: () => T) = {
+  protected def retryWhen[T](pred: Predicate[Throwable])(op: () => T): Future[T] = {
```

Comment:
Ah good catch, it's not being used anywhere. I was using it earlier but refactored code such that it's no longer needed. I can just remove it.